ChangeLog.txt

```diff
@@ -328,6 +328,8 @@ MAP/HISTORY:
 - Added Claudius, a Hispano-Roman Nicene dux of Lusitania in the late VI century
 - Added one historical dux of Libya
 - Changed non-existant "berber" culture provinces in Sahel to Tuareg and religion from African to Central African (Berber)
+- Fixed the culture of the Astures leader and added a successor to avoid game over
+- Added the Cantabrian "Runcones" tribe as an independent duchy after 507
```

Comment:
Are the Runcones Cantanrian or Asturian?

Reply:
They are Asturian.
My bad, I think I was doing something to the Cantabrians when I wrote the changelog and got them messed up, I'll change it ASAP.